packages/synapse-react-client/src/lib/components/styled/LeftRightPanel.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Box, BoxProps, Paper, PaperProps, styled } from '@mui/material'
+import { StyledComponent } from '@emotion/styled'
+
+export const StyledOuterContainer: StyledComponent<BoxProps> = styled(Box, {
+  label: 'StyledOuterContainer',
+})(() => ({
+  minHeight: '100vh',
+  paddingTop: '50px',
+  paddingBottom: '50px',
+  background:
+    "linear-gradient(rgba(255,255,255,0), rgba(255,255,255,0)), url('https://s3.amazonaws.com/static.synapse.org/images/SynapseLoginPageBackground.svg')",
```

Comment:
This is a different alpha value than the one in SageAccountWeb. Perhaps we can add this string to the theme to be configurable per-app?